src/prost/cosmos.auth.v1beta1.rs

```diff
@@ -446,6 +492,14 @@ pub mod query_server {
             &self,
             request: tonic::Request<super::QueryModuleAccountsRequest>,
         ) -> Result<tonic::Response<super::QueryModuleAccountsResponse>, tonic::Status>;
+        /// ModuleAccountByName returns the module account info by module name
+        async fn module_account_by_name(
+            &self,
+            request: tonic::Request<super::QueryModuleAccountByNameRequest>,
+        ) -> Result<
+            tonic::Response<super::QueryModuleAccountByNameResponse>,
+            tonic::Status,
+        >;
```

Comment:
Adding the new method to a trait makes it a breaking change on the server side.

Make sure to bump the minor version in `Cargo.toml`, and perhaps record this change in `.changelog/breaking-changes`?